openmessaging-api/src/main/java/io/openmessaging/api/Message.java

```diff
@@ -259,6 +259,19 @@ public TopicPartition getTopicPartition() {
         return new TopicPartition(topic, getSystemProperties(SystemPropKey.PARTITION));
     }
 
+    public int getPriorityLevel() {
+        String pro = this.getSystemProperties(SystemPropKey.PRIORITY);
+        if (pro != null) {
+            return Integer.parseInt(pro);
+        }
+
+        return -1;
```

Comment:
It is recommended to make -1  as a constant

Reply:
The new commit's used a constant NOOP_PRIORITY_LEVEL instead of -1